packages/components/src/item-group/stories/index.story.tsx

```diff
@@ -67,6 +69,8 @@ export const NonClickableItems: StoryFn< typeof ItemGroup > = Template.bind(
 	{}
 );
 NonClickableItems.args = {
+	isBordered: true,
+	isSeparated: true,
```

Comment:
Nit: for consistency with how we usually write the stories: 

```suggestion
	...Default.args,
```

Reply:
Sounds good, done in 5e26b6fe8939cc87cabc95c368eaca1757e24074